src/rgw/rgw_zone_types.h

```diff
@@ -555,20 +601,27 @@ struct RGWZoneGroupPlacementTier {
 
   bool allow_read_through = false;
   uint64_t read_through_restore_days = 1;
+  std::string restore_storage_class = RGW_STORAGE_CLASS_STANDARD;
+
+  RGWZoneGroupTierS3Glacier s3_glacier;
 
   int update_params(const JSONFormattable& config);
   int clear_params(const JSONFormattable& config);
 
   void encode(bufferlist& bl) const {
-    ENCODE_START(3, 1, bl);
+    ENCODE_START(4, 1, bl);
```

Comment:
@soumyakoduri @thotz please note the fix for this in https://github.com/ceph/ceph/pull/62515 (has this been backported?)

Reply:
No. This feature is not present in squid upstream release.  (Yet to be backported into downstream though.)

Reply:
> Yet to be backported into downstream though

okay, thanks - just wanted to make sure we don't miss the fix downstream

Reply:
yes. Thank you! We weren't aware of this issue till now.